src/fate.ts

```diff
@@ -10,7 +10,7 @@ export class Fate<T = unknown> {
 	public data: T | null;
 	public readonly errorLog: FateObject['errorLog'];
 	public readonly messageLog: FateObject['messageLog'];
-	public readonly errorThreshold: FateObject['errorThreshold'];
+	public errorThreshold: FateObject['errorThreshold'];
```

Comment:
We can make this property `readonly` and change the type to `UInt` to get the type and value guarantee of `strong-types`.